doc/classes/CanvasItem.xml

```diff
@@ -417,6 +417,13 @@
 				Returns the transform from the coordinate system of the canvas, this item is in, to the [Viewport]s coordinate system.
 			</description>
 		</method>
+		<method name="get_effective_z_index" qualifiers="const">
+			<return type="int" />
+			<description>
+				Returns the total Z index, accounting for [member z_as_relative].
+				This does not account for other effects such as [member top_level], [member y_sort_enabled], or sorting by hierarchy (see [member get_index].)
```

Comment:
```suggestion
				This does not account for other effects such as [member top_level], [member y_sort_enabled], or sorting by hierarchy (see [member get_index]).
```

Reply:
This was intended to refer to `Node.get\_index`, unsure how to fix that.

Reply:
```suggestion
				This does not account for other effects such as [member top_level], [member y_sort_enabled], or sorting by hierarchy (see [method Node.get_index]).
```

Reply:
Tested and applied.

Reply:
Should this conversation be marked as resolved?